client/blocks/login/social.jsx

```diff
@@ -110,6 +119,11 @@ class SocialLoginForm extends Component {
 	}
 
 	render() {
+		// TODO: uncomment before merge and remove next line
+		//const isPopup = window.opener && window.opener !== window;
+		const isPopup = true;
+		const redirectUri = isPopup ? login( { isNative: true, socialService: 'google' } ) : null;
```

Comment:
We'll have to test if adding a `redirectTo` param here will work fine with this flow